packages/ui/src/multiplying-architecture/KaotoEditorChannelApi.ts

```diff
@@ -1,5 +1,15 @@
 import { KogitoEditorChannelApi } from '@kie-tools-core/editor/dist/api';
+import { AbstractSettingsAdapter } from '../models/settings';
 
 export interface KaotoEditorChannelApi extends KogitoEditorChannelApi {
+  /**
+   * @deprecated Use `getVSCodeKaotoSettings` instead
+   * Returns the URL of the catalog.
+   */
   getCatalogURL(): Promise<string | undefined>;
```

Comment:
@apupier, I think we can remove this method once 2.2.0 is released, as we're gonna use the `SettingsAdapter` to pass all settings.